src/ezdxf/addons/drawing/pymupdf.py

```diff
@@ -21,13 +22,13 @@
 from . import layout, recorder
 
 is_pymupdf_installed = True
+pymupdf: Any = None
 try:
-    import fitz
+    import pymupdf  # type: ignore[import-untyped]
```

Comment:
fitz is a legacy name and mupdf is the new name: [source](https://pymupdf.readthedocs.io/en/latest/intro.html#note-on-the-legacy-module-name-fitz)